1_wqpdata/log/tasks_1_wqp_inventory.ind

```diff
@@ -0,0 +1,2 @@
+indication_time: 2018-06-07 18:16:16 UTC
```

Comment:
I recently added to scipiper so that it can store more information in this file (one hash per task) (https://github.com/USGS-R/scipiper/pull/44/commits/fa04118deb37a2dea6336bf0d686f90e3df8f868). I think when you update it'll be useful to add `ind_complete=TRUE` in calls to `create_task_makefile`. Probably a good idea to update before pulling much more data.